packages/distance-sensor/src/distance-sensor.ts

```diff
@@ -0,0 +1,34 @@
+import {ExpansionHub} from "@rev-robotics/expansion-hub";
+import { VL53L0X } from "./drivers/vl53l0x";
+
+export class DistanceSensor {
+    constructor(hub: ExpansionHub, channel: number) {
+        this.device = new VL53L0X(hub, channel);
+    }
+
+    private readonly device: DistanceSensorDriver
+    private timer?: NodeJS.Timer
+
+    async setup() {
+        await this.device.setup();
+    }
+
+    async getDistanceMillimeters(): Promise<number> {
+        return await this.device.getDistanceMillimeters();
+    }
+
+    startMeasuringDistance(onDistanceRecorded: (_: number) => void,
```

Comment:
The parameter passed to `onDistanceRecorded()` should have a name that indicates the unit.

Reply:
Added parameter name and mentioned units in documentation